index.html

```diff
@@ -209,6 +209,23 @@ <h3 class="skill-language">Skills</h3>
       </div>
     </section>
   </footer>
+
+  <section class="formx">
+    <div class="contact-form">
+      <h2 class="contact-title">Contact me</h2>
+      <p class="contact-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it </p>
+      <div class="form">
+        <form action="https://formspree.io/f/mjvzwnzn" method="POST">
```

Comment:
According to MDN doc provided by microverse in the course work, Is advices to use ul or li in out forms to provide accessibility.
🥳 👎  Good job in implementing the Figma UI which matches you project Kudoz.